skrf/io/touchstone.py

```diff
@@ -149,7 +152,7 @@ def __init__(self, file: typing.Union[str, typing.TextIO],
         self.gamma = []
         self.z0 = []
 
-        if self.is_from_hfss:
+        if self.hfss_port_impedances:
```

Comment:
I think this name is a bit misleading for a Boolean. `has_hfss_port_impedances` ?